packages/store-indexer/bin/parseEnv.ts

```diff
@@ -9,22 +8,30 @@ const commonSchema = z.intersection(
     MAX_BLOCK_RANGE: z.coerce.bigint().positive().default(1000n),
     POLLING_INTERVAL: z.coerce.number().positive().default(1000),
   }),
-  z
-    .object({
+  z.union([
+    z.object({
       RPC_HTTP_URL: z.string(),
+      RPC_WS_URL: z.string().optional(),
+    }),
+    z.object({
+      RPC_HTTP_URL: z.string().optional(),
       RPC_WS_URL: z.string(),
-    })
-    .partial()
-    .refine((values) => Object.values(values).some(isDefined))
```

Comment:
refine wasn't surfacing nice error messages, would need superRefine for that and didn't feel like wiring that up, so I switched this to a union